### PR TITLE
code cleanup for cmd/kubeadm

### PR DESCRIPTION
**Pull request by lojies**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
remove repeated pkg import and Omit comparison with boolean constant
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:28]: instead, the `"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"` can be removed.

File: cmd/kubeadm/app/componentconfigs/kubelet.go [hunk 2]
```diff
@@ -163,7 +163,7 @@ func (kc *kubeletConfig) Default(cfg *kubeadmapi.ClusterConfiguration, _ *kubead
 
 	if kc.config.Authentication.Anonymous.Enabled == nil {
 		kc.config.Authentication.Anonymous.Enabled = utilpointer.BoolPtr(kubeletAuthenticationAnonymousEnabled)
-	} else if *kc.config.Authentication.Anonymous.Enabled != kubeletAuthenticationAnonymousEnabled {
+	} else if *kc.config.Authentication.Anonymous.Enabled {
```
Comment (neolit123) [inline cmd/kubeadm/app/componentconfigs/kubelet.go:166]: i don't see keeping these comparisons with the constants / defaults as a problem.
is this something a static check tool reports as a problem?

File: cmd/kubeadm/app/componentconfigs/kubelet.go [hunk 3]
```diff
@@ -178,7 +178,7 @@ func (kc *kubeletConfig) Default(cfg *kubeadmapi.ClusterConfiguration, _ *kubead
 	// Let clients using other authentication methods like ServiceAccount tokens also access the kubelet API
 	if kc.config.Authentication.Webhook.Enabled == nil {
 		kc.config.Authentication.Webhook.Enabled = utilpointer.BoolPtr(kubeletAuthenticationWebhookEnabled)
-	} else if *kc.config.Authentication.Webhook.Enabled != kubeletAuthenticationWebhookEnabled {
+	} else if !*kc.config.Authentication.Webhook.Enabled {
```
Comment (neolit123) [inline cmd/kubeadm/app/componentconfigs/kubelet.go:181]: [1]

File: cmd/kubeadm/app/componentconfigs/kubelet.go (hunk 2 above)
Comment (lojies) [reply cmd/kubeadm/app/componentconfigs/kubelet.go:166]: yes, static check tool  thinks should omit comparison with boolean constant, and it reports this as a problem.
https://staticcheck.io/docs/checks#S1002

Comment (neolit123) [reply cmd/kubeadm/app/componentconfigs/kubelet.go:166]: ok, this cleanup seems fine for the sake of not having follow up PRs that do the same cleanup.

File: cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go (hunk 1 above)
Comment (lojies) [reply cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:28]: thanks for the suggestion, i have changed this.

File: cmd/kubeadm/app/componentconfigs/kubelet.go (hunk 3 above)
Comment (fabriziopandini) [reply cmd/kubeadm/app/componentconfigs/kubelet.go:181]: Should this be `if *kc.config.Authentication.Webhook.Enabled` ?

Comment (fabriziopandini) [inline cmd/kubeadm/app/componentconfigs/kubelet.go:181]: Should this be if *kc.config.Authentication.Webhook.Enabled ?

Comment (lojies) [reply cmd/kubeadm/app/componentconfigs/kubelet.go:181]: // kubeletAuthenticationWebhookEnabled set the default value to enable authentication webhook
	kubeletAuthenticationWebhookEnabled = true
        if *kc.config.Authentication.Webhook.Enabled != kubeletAuthenticationWebhookEnabled
==   if *kc.config.Authentication.Webhook.Enabled != true
==   if !*kc.config.Authentication.Webhook.Enabled

**Pull request merged**